assets/xml/objects/object_link_child.xml

```diff
@@ -122,35 +134,60 @@
-        <Limb Name="object_link_child_Lodlimb_01E190" Type="LOD" EnumName="OBJECT_LINK_CHILD_LIMB_10" Offset="0x1E190" />
-        <Limb Name="object_link_child_Lodlimb_01E1A0" Type="LOD" EnumName="OBJECT_LINK_CHILD_LIMB_11" Offset="0x1E1A0" />
-        <Limb Name="object_link_child_Lodlimb_01E1B0" Type="LOD" EnumName="OBJECT_LINK_CHILD_LIMB_12" Offset="0x1E1B0" />
-        <Limb Name="object_link_child_Lodlimb_01E1C0" Type="LOD" EnumName="OBJECT_LINK_CHILD_LIMB_13" Offset="0x1E1C0" />
-        <Limb Name="object_link_child_Lodlimb_01E1D0" Type="LOD" EnumName="OBJECT_LINK_CHILD_LIMB_14" Offset="0x1E1D0" />
-        <Limb Name="object_link_child_Lodlimb_01E1E0" Type="LOD" EnumName="OBJECT_LINK_CHILD_LIMB_15" Offset="0x1E1E0" />
-        <Skeleton Name="object_link_child_Skel_01E244" Type="Flex" LimbType="Standard" LimbNone="OBJECT_LINK_CHILD_LIMB_NONE" LimbMax="OBJECT_LINK_CHILD_LIMB_MAX" EnumName="ObjectLinkChildLimb" Offset="0x1E244" />
+
+        <DList Name="gLinkHumanSheathEmptyDL" Offset="0x1DA90" />
+
+        <Mtx Name="gLinkHumanSheathedKokiriSwordMtx" Offset="0x1DA98" />
+        <DList Name="gLinkHumanSheathedKokiriSwordDL" Offset="0x1DAD8" />
+
+        <Mtx Name="gLinkHumanSheathedRazorSwordMtx" Offset="0x1DB00" />
+        <DList Name="gLinkHumanSheathedRazorSwordDL" Offset="0x1DB40" />
+        <DList Name="gLinkHumanSheathedGildedSwordDL" Offset="0x1DB60" />
+        <Mtx Name="gLinkHumanHylianShieldMtx" Offset="0x1DB88" />
+        <DList Name="gLinkHumanHylianShieldWithMtxDL" Offset="0x1DBC8" />
+        <Mtx Name="gLinkHumanMirrorShieldMtx" Offset="0x1DBD8" />
+        <DList Name="gLinkHumanMirrorShieldWithMtxDL" Offset="0x1DC18" />
+
+        <DList Name="gLinkHumanRightHandHoldingHylianShieldDL" Offset="0x1DC28" />
+        <DList Name="object_link_child_DL_01DC38" Offset="0x1DC38" />
+        <DList Name="gLinkHumanRightHandHoldingMirrorShieldDL" Offset="0x1DC48" />
+        <DList Name="object_link_child_DL_01DC58" Offset="0x1DC58" />
+        <DList Name="gLinkHumanLeftHandHoldingKokiriSwordDL" Offset="0x1DC68" />
+        <DList Name="object_link_child_DL_01DC78" Offset="0x1DC78" />
+        <DList Name="gLinkHumanLeftHandHoldingRazorSwordDL" Offset="0x1DC88" />
+        <DList Name="object_link_child_DL_01DC98" Offset="0x1DC98" />
+        <DList Name="gLinkHumanLeftHandHoldingGildedSwordDL" Offset="0x1DCA8" />
+        <DList Name="object_link_child_DL_01DCC0" Offset="0x1DCC0" />
+        <DList Name="gLinkHumanLeftHandHoldingGreatFairysSwordDL" Offset="0x1DCD8" />
+        <DList Name="object_link_child_DL_01DCE8" Offset="0x1DCE8" />
+        <DList Name="gLinkHumanRightHandHoldingHookshotDL" Offset="0x1DCF8" />
+        <DList Name="object_link_child_DL_01DD08" Offset="0x1DD08" />
+        <DList Name="gLinkHumanRightHandHoldingBowDL" Offset="0x1DD18" />
+        <DList Name="object_link_child_DL_01DD28" Offset="0x1DD28" />
```

Comment:
Yes. Many of those end in `gsSPBranchList` instead of a normal `gsSPEndDisplayList` so Z64Utils doesn't like it.
Btw this PR doesn't intends to document the whole player objects yet